src/Components/Web.JS/src/Rendering/BrowserRenderer.ts

```diff
@@ -1,15 +1,11 @@
 import { RenderBatch, ArrayBuilderSegment, RenderTreeEdit, RenderTreeFrame, EditType, FrameType, ArrayValues } from './RenderBatch/RenderBatch';
```

Comment:
You'll see that lots of the event-related stuff was decoupled from `BrowserRenderer` here. Mostly the spaghettification was not for any good reason and just evolved there as we added more and more event-related features over time.